### ISSUE TITLE
Yaltai Output Format Inconsistency: JSON Content in XML Files

### ISSUE THREAD

**Issue by TheoMoins** [PonteIneptique/rtk#9]
Hi ! 

While testing the code on the provided example, I encountered an issue at the YALTAi step. I noticed that at this stage, the outputs are written in JSON format but saved in XML files.

This inconsistency between the file content and extension caused issues in the workflow (not running the rest of the process).

I resolved the problem by adding the `--alto` option to the Yaltai command. After this modification, everything works as expected but I think it's weird that I had to do that...

Is this behavior normal or intended?
Could this be due to a recent change in Yaltai or Kraken?

### Environment

Yaltai version on yaltaienv: 2.0.4
Kraken version on krakenv: 5.2.9

**Comment by TheoMoins** [comment]
Sorry I think I made a mistake on informing the wrong binary file for kraken and yaltai.

**Comment by PonteIneptique** [comment]
I am pretty sure you found a bug due to the latest release of YALTAi :) Ill check it :)

**Comment by TheoMoins** [comment]
I've identified another issue, probably related to the recent YALTAi update.. 

It seems that the `pbar_parsing` function in the class `YALTAiCommand` no longer works correctly, as the output log format has changed.

The previous regular expression was based on the log format that included "kraken_yaltai" at the end :

```python
return re.findall(r"Serializing as alto into (.+\.xml)\s+kraken_yaltai", stdout)
```

The new log format is different in my setup:
```
INFO     Serializing as alto into bn-f-departement-des-manuscrits-naf-5094/f9-2v.xml                         krakn.py:63
```

I quickly modified the regular expression to match the new format:
```python
return re.findall(r"Serializing as alto into (.+\.xml)\s+", stdout)
```

The function works correctly and files are properly added to the outputs :)

**Comment by PonteIneptique** [comment]
I have a very heavy week of teaching but we'll get to it on monday. You are right, every bug you found is tied to the newest version of YALTAi.

**Comment by PonteIneptique** [comment]
Thanks for the heads up, I was able to fix it in #10 :)
